openapi/stripe/types.bal

```diff
@@ -59,35 +66,35 @@ public type Customer record {
     # The customer's email address.
     string? email?;
     # Unique identifier for the object.
-    string? id;
+    string id;
     # The prefix for the customer used to generate unique invoice numbers.
     string? invoice_prefix?;
-    InvoiceSettingCustomerSetting? invoice_settings?;
+    InvoiceSettingCustomerSetting invoice_settings?;
     # Has the value `true` if the object exists in live mode or the value `false` if the object exists in test mode.
-    boolean? livemode;
+    boolean livemode;
     # Set of [key-value pairs](https://stripe.com/docs/api/metadata) that you can attach to an object. This can be useful for storing additional information about the object in a structured format.
     record {} metadata?;
     # The customer's full name or business name.
     string? name?;
     # The suffix of the customer's next invoice number, e.g., 0001.
-    int? next_invoice_sequence?;
+    int next_invoice_sequence?;
     # String representing the object's type. Objects of the same type share the same value.
-    string? 'object;
+    string 'object;
     # The customer's phone number.
     string? phone?;
     # The customer's preferred locales (languages), ordered by preference.
     string[]? preferred_locales?;
     # Mailing and shipping address for the customer. Appears on invoices emailed to this customer.
     Shipping? shipping?;
     # The customer's payment sources, if any.
-    ApmsSourcesSourceList? sources?;
+    ApmsSourcesSourceList sources?;
     # The customer's current subscriptions, if any.
-    SubscriptionList? subscriptions?;
-    CustomerTax? tax?;
```

Comment:
It seems all the nullable types are removed with the changes. So I guess the previous generation was done using the `--nullable` flag in the openAPI command. 
So lets try generating the client using the same flag.